packages/dds/tree/src/test/feature-libraries/optional-field/optionalChangeRebaser.test.ts

```diff
@@ -501,7 +522,7 @@ export function testRebaserAxioms() {
 				{
 					numberOfEditsToRebase: 3,
 					numberOfEditsToRebaseOver: isStress ? 5 : 3,
-					numberOfEditsToVerifyAssociativity: isStress ? 6 : 4,
+					numberOfEditsToVerifyAssociativity: isStress ? 6 : 3,
```

Comment:
are there bugs here with 4?

Reply:
I just checked and it passes with 4. I assume Yann just wanted to reduce the test time.